ai-ml-samples/interactive/ImageClassificationInSpark.ipynb

```diff
@@ -0,0 +1,280 @@
+{
```

Comment:
<p>Please add the Workbench and Github widgets as seen here: <a href="https://github.com/GoogleCloudPlatform/generative-ai/blob/main/gemini/getting-started/intro_gemini_1_5_pro.ipynb" target="_blank">https://github.com/GoogleCloudPlatform/generative-ai/blob/main/gemini/getting-started/intro_gemini_1_5_pro.ipynb</a></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudDataproc/cloud-dataproc/pull/166/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
<p>Added the widgets. With workbench, user would need to manually import the notebook as it won't open directly into remote s8s notebook but at least user would land on page to create new workbench instance.</p>